build/stf-run-ci/tasks/setup_stf_local_build.yml

```diff
@@ -36,6 +42,9 @@
 - name: Load Smart Gateway Operator CSV
   shell: oc apply -f working/smart-gateway-operator-bundle/manifests/smart-gateway-operator.clusterserviceversion.yaml -n "{{ namespace }}"
 
+- name: Revert local change to role_binding.yaml
```

Comment:
Oh thanks, this will save me some minutes in the future I'm sure :)

Reply:
Bah! I put it in the wrong spot :) This needs to be done after loading the STO RBAC...